lesson1/index.html

```diff
@@ -1,17 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="style.css" type="text/css" />
-
-    <title>TerraceTechフロントエンドエンジニア養成所</title>
-  </head>
-  <body>
-    <div>
-      <ul id="target"></ul>
-    </div>
-  </body>
-  <script src="index.js" async></script>
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link rel="stylesheet" href="style.css" type="text/css" />
+        <script src="index.js" async></script>
+        <title>TerraceTechフロントエンドエンジニア養成所</title>
+    </head>
+    <body>
+        <div>
+            <ul id="target"></ul>
```

Comment:
IMO
It would be good to add a js-** prefix to indicate that the id is used in JavaScript.

ex.) id=“js-target"

Reply:
Thank you for your comment.
I will rename the id.